src/resultspage.cpp

```diff
@@ -112,6 +122,10 @@ ResultsPage::ResultsPage(PerfParser* parser, QWidget* parent)
     m_summaryPageDock->addDockWidgetAsTab(m_disassemblyDock, KDDockWidgets::InitialVisibilityOption::StartHidden);
     m_disassemblyDock->toggleAction()->setEnabled(false);
     m_summaryPageDock->setAsCurrentTab();
+#if KChart_FOUND
+    m_frequencyDock = dockify(m_frequencyPage, QStringLiteral("frequency"), tr("&Frequency"), tr("Ctrl+F"));
```

Comment:
ctrl + f should be reserved for "find" , please use a different shortcut

Reply:
I changed it to ctrl+e